pkg/network/setup/podnic.go

```diff
@@ -319,7 +366,11 @@ func (l *podNIC) storeCachedDomainIface(domainIface api.Interface) error {
 }
 
 func generateInPodBridgeInterfaceName(podInterfaceName string) string {
-	return fmt.Sprintf("k6t-%s", podInterfaceName)
+	bridgeName := fmt.Sprintf("k6t-%s", podInterfaceName)
+	if len(bridgeName) > namescheme.MaxIfaceNameLen {
```

Comment:
What change in this commit/PR triggers the need to this change.
If this is just an existing bug fix, please introduce it in a separate PR, this one is big enough.

Reply:
This was meaningful for the iface rename changes; we can live without this until we support unplug. 

Good catch, thanks.

Reply:
Done.